Dockerfile

```diff
@@ -1,31 +1 @@
-FROM amazonlinux:2
-
-WORKDIR /usr/src/extension
-
-COPY . .
-
-RUN yum install -y \
-    gzip \
-    tar
-
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash && \
-    . ~/.nvm/nvm.sh && \
-    nvm install 17 && \
-    npm install yarn -g && \
-    yarn install  && \
```

Comment:
I just realized that this is an open source repo and we'll need to leave in the yarn install line so the dependency tracking feature works for now.

Reply:
Need to add the remaining steps [here](https://github.com/Workiva/vs-code-format-on-save/pull/26/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L19) as well as aviary was getting failed due to security issues.